proto/regen/ecocredit/v1alpha1/tx.proto

```diff
@@ -274,4 +280,41 @@ message MsgUpdateClassMetadata {
 }
 
 // MsgUpdateClassMetadataResponse is the MsgUpdateClassMetadata response type.
-message MsgUpdateClassMetadataResponse {}
\ No newline at end of file
+message MsgUpdateClassMetadataResponse {}
+
+message MsgSell {
+  string owner = 1;
+  repeated Order orders = 2;
+
+  message Order {
+    string batch_denom = 1;
+    string quantity = 2;
+    string ask_denom = 3;
+    string ask_price = 4;
```

Comment:
Could this just be one `Coin` field `ask_coin` ?

Reply:
Sure 👍

Reply:
I'm assuming that the `ask_coin` is the price per one unit right?

Reply:
yes that's what my assumption would be.

Reply:
let's add a doc string to specify the details.